### DISCUSSION TITLE
Plugins for other P2P networks

### DISCUSSION THREAD

**Discussion by tandy-1000** [spacedriveapp/spacedrive#73] [Feature Request]
It would be super cool to be able to view Soulseek shares from spacedrive, this could be implemented using [soulseek.net](https://github.com/jpdillingham/Soulseek.NET).

Anyone have any ideas for other potential integrations?

**Comment by hecaptnkya** [comment]
Im all ears..

**Comment by ghost** [comment]
[Jedi Archives](https://git.disroot.org/hirrolot19/jedi-archives)
